sql/src/main/java/io/crate/operation/collect/sources/InformationSchemaIterables.java

```diff
@@ -64,6 +66,10 @@ public InformationSchemaIterables(final Schemas schemas,
         RoutineInfos routineInfos = new RoutineInfos(ftResolver, clusterService);
         routines = FluentIterable.from(routineInfos).filter(Objects::nonNull);
         sqlFeatures = new SqlFeaturesIterable();
+        KeyColumnUsageInfos keyColumnUsageInfos = new KeyColumnUsageInfos();
+        keyColumnUsages = FluentIterable.from(keyColumnUsageInfos);
```

Comment:
pls replace with
```keyColumnUsages = FluentIterable.from(Collections.emptyIterator()); ```
and delete the `KeyColumnUsageInfos` class as it is useless ;)